### PR TITLE
[CONTENT] Refactoring: Absorb tutorial-like references and content from crate-clients-tools

### PR DESCRIPTION
**Pull request by amotl**
## About

Previously, a bit of tutorial-like material, mostly in form of references/links, has been slotted into crate-clients-tools as an interim solution. Let's have them play on the main stage now.

## Status

This patch is merely a content transfer / refactoring, without adding any style yet. **The outcome currently has many _walls of links_, which is unpleasant.** Sure enough, more can be done on behalf of subsequent iterations, in order to provide better appearance and guidance. For this patch, we can be happy enough that the content finally found the right slot.

## Preview

- https://cratedb-guide--29.org.readthedocs.build/getting-started/connect.html
- https://cratedb-guide--29.org.readthedocs.build/integrate/index.html
- https://cratedb-guide--29.org.readthedocs.build/topic/analysis/index.html
- https://cratedb-guide--29.org.readthedocs.build/topic/ml/index.html

## Backlog
When integrating this patch, this other one can be merged afterwards, in order to conclude the content migration.
- https://github.com/crate/crate-clients-tools/pull/82

/cc @karynzv, @marijaselakovic, @matkuliak, @hammerhead, @hlcianfagna, @surister, @proddata, @ckurze, @stephanec76

### CODE REVIEW

**Review by seut — COMMENTED** [review]
👍 Added some questions and suggestions. I think the menu structure isn't good as now, see my related comment about integrations vs topics.

File: docs/index.md [hunk 1]
```diff
@@ -157,6 +157,7 @@ reference-architectures/index
 topic/analysis/index
 topic/timeseries/index
 topic/ml/index
+topic/testing
```
[the remark this answers is not in the document]
Comment (amotl) [reply docs/index.md:160]: Sure, we can have a look how this looks like, independently of semantic matters. Thanks.

File: docs/topic/analysis/grafana.rst [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+CrateDB Admin UI which is the admin UI of your cluster. When accessing it for
+the first time, you will need the username and password that you set when you
+deployed the cluster.
+
+.. image:: ../../_assets/img/integrations/grafana/grafana-admin-create-table.png
+   :alt: Creating table in Admin UI
+
+After executing the second SQL statement, the "nyc_taxi" table will be
+populated with data. Depending on your cluster configuration this can take
+around 40 minutes.
```
[the remark this answers is not in the document]
Comment (amotl) [reply docs/topic/analysis/grafana.rst:57]: I agree. However, I am not touching the content too much on this iteration, so we need to carry this over into a subsequent iteration. Thanks for spotting!

File: docs/topic/analysis/index.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
+(analysis)=
+(visualization)=
+
+# Data Analysis and Visualization
+
+This section of the documentation covers integrations of CrateDB with other
```
[the remark this answers is not in the document]
Comment (amotl) [reply docs/topic/analysis/index.md:6]: This is right on the spot! I've certainly also identified this as a [smell](https://en.wikipedia.org/wiki/Code_smell), but continued to bring in the content first. On another iteration here, I will adjust the page layout to reflect the structure of https://cratedb.com/docs/clients/ as the baseline for further discussions. Thanks again!

File: docs/getting-started/connect.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  - [SQLAlchemy Support], [SQLAlchemy by Example], [SQLAlchemy Code Examples]
+- pandas and Dask
+  - [Importing Parquet files into CrateDB using Apache Arrow and SQLAlchemy]
+  - [Guide to efficient data ingestion to CrateDB with pandas]
+  - [Guide to efficient data ingestion to CrateDB with pandas and Dask]
+  - [Efficient batch/bulk INSERT operations with pandas, Dask, and SQLAlchemy]
+  - [pandas code examples]
+  - [Dask code examples]
+- Polars
+  - [Polars code examples]
```
[the remark this answers is not in the document]
Comment (amotl) [reply docs/getting-started/connect.md:29]: I will see what I can do here while compressing the "Getting started" section into a single page, as suggested. Thanks.

[the remark this answers is not in the document]
Comment (amotl) [reply docs/getting-started/connect.md:29]: Corresponding improvements have been implemented on behalf of a stacked PR.

- GH-35

File: docs/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (amotl) [reply docs/index.md:160]: Implemented on behalf of a stacked PR.

- GH-34

File: docs/topic/analysis/grafana.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (amotl) [reply docs/topic/analysis/grafana.rst:57]: Diverted into an issue, thanks again.

- GH-36

File: docs/topic/analysis/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (amotl) [reply docs/topic/analysis/index.md:6]: Layout/shape/structure has been improved with d1c46cadf and 2b4b45deb938 on behalf of the other PR.

- GH-34

**Review by seut — APPROVED** [review]
👍

**Pull request merged**